access_amherst_backend/access_amherst_algo/rss_scraper/parse_rss.py

```diff
@@ -112,10 +113,21 @@
     },
 }
 
+# # Configure logging
+# logging.basicConfig(
+#     level=logging.INFO,
+#     format='%(asctime)s - %(levelname)s - %(message)s'
+# )
```

Comment:
Please get rid of these comments since we are using the config you mentioned right below this!